lib/backpex/adapters/ash.ex

```diff
@@ -75,8 +75,17 @@ if Code.ensure_loaded?(Ash) do
     Deletes multiple items.
     """
     @impl Backpex.Adapter
-    def delete_all(_items, _config) do
-      raise "not implemented yet"
+    def delete_all(items, live_resource) do
+      config = live_resource.config(:adapter_config)
+      primary_key = live_resource.config(:primary_key)
+
+      ids = Enum.map(items, &Map.fetch!(&1, primary_key))
+
+      config[:resource]
+      |> Ash.Query.filter(^Ash.Expr.ref(primary_key) in ^ids)
+      |> Ash.bulk_destroy!(:destroy, %{})
+
+      :ok
```

Comment:
How do we want to catch errors? Currently, the item action, e.g., Delete, uses `try rescue` and pattern matches on the error. I don't like this, as the item action itself should not figure out why the deletion failed. In the case of Ash, the bulk delete might contain errors (in the result struct), even if it did not raise an error. I suggest moving the error handling to the adapter.  @pehbehbeh

Reply:
We decided to use `{:ok, deleted_items}` for now. We plan to improve error handling in the future so that the item action can match on `:ok` or `:error` without the need to use `try ... rescue`